srtcore/common.cpp

```diff
@@ -179,32 +177,6 @@ void CTimer::tick()
 }
 
 
-void CTimer::triggerEvent()
-{
-    pthread_cond_signal(&m_EventCond);
-}
-
-CTimer::EWait CTimer::waitForEvent()
-{
-    timeval now;
-    timespec timeout;
-    gettimeofday(&now, 0);
-    if (now.tv_usec < 990000)
-    {
-        timeout.tv_sec = now.tv_sec;
-        timeout.tv_nsec = (now.tv_usec + 10000) * 1000;
-    }
-    else
-    {
-        timeout.tv_sec = now.tv_sec + 1;
-        timeout.tv_nsec = (now.tv_usec + 10000 - 1000000) * 1000;
-    }
-    enterCS(m_EventLock);
-    int reason = pthread_cond_timedwait(&m_EventCond, &m_EventLock.ref(), &timeout);
-    leaveCS(m_EventLock);
-
-    return reason == ETIMEDOUT ? WT_TIMEOUT : reason == 0 ? WT_EVENT : WT_ERROR;
```

Comment:
These are possible error codes that `pthread_cond_timedwait` can return. We don't need to handle them once the core uses the right cvs and mutexes.

The pthread_cond_timedwait() and pthread_cond_wait() functions may fail if: 
**EINVAL** 
The value specified by cond, mutex, or abstime is invalid. 
**EINVAL** 
Different mutexes were supplied for concurrent pthread_cond_timedwait() or pthread_cond_wait() operations on the same condition variable. 
**EPERM** 
The mutex was not owned by the current thread at the time of the call.